docs/CI-CD/GithubActions.md

```diff
@@ -28,33 +30,65 @@ echo $He_SP_PWD_CICD
 
 ```
 
-## Add Service Principal Credentials to Github Secrets
+### Create a Docker hub User account and Create Personal Access Token
+
+ADD STEPS HERE
+
+### Set up Secrets in GitHub Action workflows
+
+[GitHub Secrets](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets) are encrypted and allow you to store sensitive information, such as access tokens, in your repository.
+
+You could use GitHub secrets to store your Azure Credentials, Publish profile of your Web app, container registry credentials or any such sensitive details which are required to automate your CI/CD workflows using GitHub Actions.
+
+#### Creating secrets
 
-In the browser, navigate to your forked repo page in Github to set up credentials with the newly created service principal.
+1. On GitHub, navigate to the main page of the repository.
+1. Under your repository name, click on the "Settings" tab.
+1. In the left sidebar, click Secrets.
+1. On the right bar, click on "Add a new secret"
+   ![alt text](../images/create-secret.png "GitHub Secrets")
+1. Type a name for your secret in the "Name" input box.
+1. Type the value for your secret.
+1. Click Add secret.
+   ![alt text](../images/Add-secret-name-value.png "Github Secret Details")
 
-1. Go to Settings -> Secrets
-2. Select "Add a new secret"
-3. Add the secret:
-    - Name: REGISTRY_USERNAME
-    - Value: {copy output from `echo $He_SP_ID_CICD` command}
-4. Add the secret:
-    - Name: REGISTRY_PASSWORD
-    - Value: {copy output from `echo $He_SP_PWD_CICD` command}
+The following secrets are required for the workflow to run:
```

Comment:
again - I would separate as you don't need all of these values unless you're using both and most of our customers will use one or the other